integration_tests/test_rebase.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from git_wrapper import exceptions
+from git_wrapper.rebase import GitWrapperRebase
+
+
+def test_rebase(repo_root):
+    rebase = GitWrapperRebase(repo_root)
+    branch_name = "mybranches/test_rebase"
+    rebase_to = "2e6c014bc296be90a7ed04d155ea7d9da2240bbc"  # Hash for 0.1.0 tag
+
+    assert rebase.repo.active_branch.name == "master"
+    assert rebase.repo.head.object.hexsha != rebase_to
+
+    # Create a branch based on an old tag
+    rebase.repo.git.branch(branch_name, "0.0.1")
+
+    # Rebase that branch
+    rebase.to_hash(branch_name=branch_name, hash_=rebase_to)
+    assert rebase.repo.active_branch.name == branch_name
```

Comment:
That line tests for that fix in PR28, @fuzzball81 :)

Reply:
Great!

Reply:
Great!